bin/run-test.sh

```diff
@@ -50,5 +50,3 @@ fi
 
 # Run non-interference tests
 node test/non-interference/http-e2e.js || exit 1
-node test/non-interference/express-e2e.js || exit 1
```

Comment:
How do we make sure these get added back in the future?

More generally, I would like to see rationale like this documented in the issue comment and also in the commit message. When we look back at the history we would be wondering what happened and would have no documentation of the rationale.